BREAKING.md

```diff
@@ -45,15 +44,6 @@ Previously, the fluidErrorCode value (a pascaleCased term) was often used as the
 Now all error messages can be expected to be easily-read sentences,
 sometimes followed by a colon and an inner error message when applicable.
 
-
-### Doing operations not allowed on deleted sub directory
```

Comment:
Will this move to the `Upcoming changes` section?  We should think about if/how to explain that this change was here but then was removed.  Maybe it's ok to just delete this for now though.

Reply:
This change will be done in "next" branch right?